hicexplorer/HiCMatrix.py

```diff
@@ -278,7 +282,7 @@ def getDekkerBins(self, fileName):
         self.header = []
         try:
             for line in gzip.open(fileName, 'r').readlines():
-                if line.startswith("#"):
+                if line.startswith(b"#"):
```

Comment:
Is this needed (not only on this line but in many others)? In deepTools we didn't have to do these changes when porting to python 3.